src/auro-accordion-group.js

```diff
@@ -31,10 +29,17 @@ class AuroAccordionGroup extends LitElement {
     });
   }
 
+  /**
+   * @private Internal function to update items when items are added or removed from the slot
+   */
+  handleSlotChange() {
```

Comment:
This is an interesting accommodation to the typical standard of populating the slot element. Outside of the Svelte issue, this is an approach probably needed in any element if the content of the slot is to be dynamically updated outside of the initial callback.

Reply:
This is to be documented.